README.md

```diff
@@ -66,27 +63,18 @@ Please follow [Getting Started][gs] to get your hands on Open-SIR.
 
 ## Authors
 
-* **[José Álamos][alamos]** -
-  [RIOT-OS][riot]
-* **[Felipe Huerta][huerta]** - [PhD
-  Student][phd-huerta] at [Imperial
-College London][imperial]
-* **[Sebastián Salata][salata]** - Software Engineer -
-  Full Stack
+- **[José Álamos][alamos]** - [RIOT-OS][riot]
```

Comment:
The way this was written/formatted was VERY weird. I now did it one-liner.

Reply:
much better :clap: